src/main/java/org/cryptomator/ui/addvaultwizard/CreateNewVaultNameController.java

```diff
@@ -70,6 +66,7 @@ public void back() {
 	@FXML
 	public void next() {
 		window.setScene(chooseLocationScene.get());
+		vaultName.get().trim();
```

Comment:
trim() doesn't change the original string, it returns a new one. What you want to do in order to satisfy @infeo's suggestion is this:

```suggestion
		vaultName.set(vaultName.get().trim());
```